justfile

```diff
@@ -127,15 +121,20 @@ generate cli=default_cli platform=default_platform:
         ;;
     esac
 
+# Apply the runtime.
+apply-runtime target=default_deploy_target platform=default_platform:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    kubectl apply -f ./{{ workspace_dir }}/runtime
+    if [[ {{ platform }} == "AKS-PEER-SNP" ]]; then
+        kubectl apply -f ./infra/azure-peerpods/peer-pods-config.yaml --namespace {{ target }}${namespace_suffix-}
```

Comment:
Who is supposed to place `peer-pods-config.yaml` in that directory?

Reply:
Terraform:
https://github.com/edgelesssys/contrast/blob/f6493998d65fa0c6eb4cdea46e3f64477ff7a27e/infra/azure-peerpods/main.tf#L126-L163